Form/LegacyModifySettingsForm.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Craue\ConfigBundle\Form;
+
+/**
+ * for Symfony < 2.8
+ *
+ * @author Christian Raue <christian.raue@gmail.com>
+ * @copyright 2011-2015 Christian Raue
+ * @license http://opensource.org/licenses/mit-license.php MIT License
+ */
+class LegacyModifySettingsForm extends AbstractModifySettingsForm {
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function getName() {
```

Comment:
there is a way to implement form types compatible with both 2.3 and 2.8 without deprecation notices. See https://github.com/symfony/symfony/blob/2.8/UPGRADE-2.8.md for that